pkg/registry/constants.go

```diff
@@ -20,11 +20,14 @@ const (
 	// HelmChartDefaultTag is the default tag used when storing a chart reference with no tag
 	HelmChartDefaultTag = "latest"
 
-	// HelmChartMetaMediaType is the reserved media type for Helm chart metadata
-	HelmChartMetaMediaType = "application/vnd.cncf.helm.chart.meta.v1+json"
+	// HelmChartConfigMediaType is the reserved media type for the Helm chart manifest config
+	HelmChartConfigMediaType = "application/vnd.cncf.helm.chart.config.v3+json"
```

Comment:
shouldn't it be v1 since this is the first iteration of the media type schema?

Reply:
That was my original thinking as well. There was convo in #oras about this

@stevelasker @sajayantony Can you please comment on this?

@bacongobbler please also see this list which ACR is currently using as whitelist (obviously subject to change based on Helm maintainers' decision): https://github.com/Azure/acr/blob/master/docs/artifact-media-types.json

Reply:
This is up to the helm maintainers. From a registry, we would follow what ever the helm maintainers decide is the acceptable media type. 
I think v1 also makes sense since this is the first version, but its more about aligning with helm client which might not be needed and I can see that helm4 might still have v1 or v3 which ever is currently being used.

Reply:
> helm4 might still have v1

I think this is good explanation for using v1. The schema can be versioned separately from Helm client. I've gone ahead and switched it to v1.